### ISSUE TITLE
JavaScript and/or PHP

### ISSUE THREAD

**Issue by chiaravalle** [shantimohan/XFCloudStorage#1]
Hello there.
I'd need something like this, but written in PHP or JavaScript.
Would you mind porting it?

Thanks in advance anyhow.

**Comment by shantimohan** [comment]
Unfortunately, I work only with C#.

**Comment by chiaravalle** [comment]
I see. Thanks anyhow.
Regards.

**Comment by chiaravalle** [comment]
Just for curiosity, I saw in the repo, in the XFCloudStorage/XFCloudStorage/XFCloudStorage/Services/ folder, there's just Google.
Is iCloud and OneDrive to be developed yet?

**Comment by shantimohan** [comment]
This repo was half baked. I built another project and I have successfully accessed OneDrive, DropBox and Google Drive. I planned to add that too to GitHub but got drawn into other issues. I am not yet successfully accessed iCloud. I have provisioning issues with iCloud. For now I kept aside working on iCloud. I can give you details on how to access OneDrive, DropBox and Google Drive. I have struggled a lot to get them work with C#. There is lot of support for Java and PHP also may not be that difficult.
